ocaml/xapi/cli_operations.ml

```diff
@@ -1574,6 +1574,28 @@ let vif_unplug printer rpc session_id params =
 	let force = get_bool_param params "force" in
 	(if force then Client.VIF.unplug_force else Client.VIF.unplug) rpc session_id vif
 
+let vif_configure_ipv4 printer rpc session_id params =
+	let read_optional_case_insensitive key =
```

Comment:
I don't think we need this function here (and below). The `pif_reconfigure_ip` function has one, because it wants to accept both `IP` and `ip`, and `DNS` and `dns` as arguments. That doesn't really apply here.